sklearn/ensemble/_forest.py

```diff
@@ -249,8 +249,7 @@ def decision_path(self, X):
         X = self._validate_X_predict(X)
         indicators = Parallel(n_jobs=self.n_jobs, verbose=self.verbose,
                               **_joblib_parallel_args(prefer='threads'))(
-            delayed(tree.decision_path)(X,
-                                     check_input=False)
+            delayed(tree.decision_path)(X, check_input=False)
```

Comment:
Thanks for fixing all these cosmetic issues, but it adds some noise both to the review process and to the history of this file, and it may lead to unnecessary conflicts with other pull-requests.
Therefore, we usually prefer changing only the strict minimum.

Reply:
ok to merge or do you want to undo these?

Reply:
No strong feelings, just wanted to mention it to avoid it in the future.

Reply:
that's what I figured, just wanted to confirm :)